gatsby-node.js

```diff
@@ -111,8 +111,8 @@ exports.createPages = ({ graphql, actions }) => {
   })
 }
 
-exports.modifyWebpackConfig = ({ config, stage }) => {
+exports.onCreateWebpackConfig = ({ actions, stage }) => {
```

Comment:
New [API](https://www.gatsbyjs.org/docs/migrating-from-v1-to-v2/#change-modifywebpackconfig-to-oncreatewebpackconfig). Part of the migration guide.